src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Since we now have a single TFM, can remove the 's'
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```